processing/sensorhub-process-ffmpeg/build.gradle

```diff
@@ -15,9 +15,13 @@ dependencies {
 osgi {
   manifest {
     attributes('Bundle-Vendor': 'Sensia Software LLC')
-    attributes('Bundle-Activator': 'org.sensorhub.process.video.Activator')
+    attributes('Bundle-Activator': 'org.sensorhub.impl.process.video.Activator')
     attributes('-fixupmessages': 'Classes found in the wrong directory; is:=ignore')
-    attributes('-exportcontents': 'org.sensorhub.impl.service.sos.video, org.bytedeco.ffmpeg.*')
+    attributes('-exportcontents': 'org.sensorhub.impl.service.sos.video, org.sensorhub.impl.process.video, org.bytedeco.ffmpeg')
+    attributes('Import-Package': '''
+      !javax.*, !java.*, !org.apache.*, !sun.*,
+      org.osgi.service.repository;resolution:=optional,
```

Comment:
Don't think we need to import "org.osgi.service.repository;resolution:=optional"

Reply:
No need, Ok for building and installing. Cannot test execution: c3baa89fd56ef3125e2a217bd66a55dffc815d30